File: Linguagem C/001_Lista_intro/Ex_012.c
//12) Fazer um algoritmo que receba um número representando a temperatura da água e mostre em qual estado
//físico (sólido, líquido ou gasoso) ela se encontra.
#include <stdio.h>
int main ()
{
    int temp; 

    scanf("%d", &temp);

    if(temp <= 0)
    {
        printf("Solido");
    }

    if(temp > 0 && temp < 100)
    {
        printf("Liquido");
    }

    if(temp > 100)
    {
        printf("Gasoso");
    }

    
return 0;    
}

File: Python/001_Lista_intro/Ex_012.py
"""
12) Fazer um algoritmo que receba um número representando a temperatura da  água  e mostre em qual estado 
físico (sólido, líquido ou gasoso) ela se encontra. 
"""

print("Digite a temperatura atual da agua: ")

temp = float(input())

if temp < 0:
    print("A agua esta no estado solido")
elif temp == 0:
    print("A agua possui estado solido e liquido ao mesmo tempo")
elif temp > 0 and temp <100:
    print("A agua esta em estado liquido")
elif temp == 100:
    print("A agua possui estado liquido e gasoso ao mesmo tempo")
else:
    print("A agua esta no estado gasoso")